next.config.js

```diff
@@ -1,10 +1,18 @@
 const nextTranslate = require('next-translate');
+const { ConfigSet } = require('ts-jest/dist/config/config-set');
 
 module.exports = nextTranslate({
   poweredByHeader: false,
   webpack: (config) => {
+    config.module.rules.push({
+      test: /\.svg$/i,
+      type: 'asset',
+      resourceQuery: /url/, // *.svg?url
+    });
     config.module.rules.push({
       test: /\.svg$/,
+      issuer: /\.[jt]sx?$/,
+      resourceQuery: { not: [/url/] }, // exclude react component if *.svg?url
```

Comment:
see [Use SVGR and asset SVG in the same project
](https://react-svgr.com/docs/webpack/#use-svgr-and-asset-svg-in-the-same-project)